File: 240609/학점 계산기/credit-calculator.py
N = int(input())
scores = list(map(float, input().split()))

avg = round(sum(scores) / len(scores), 1)
print(avg)
if avg >= 4.0:
    print("Perfect")
elif 3.0 <= avg < 4.0:
    print("Good")
else:
    print("Poor")

File: 251111/학점 계산기/credit-calculator.cpp
#include <iostream>
#include <vector>
#include <iomanip>
#include <numeric>
using namespace std;

int main() 
{
    // Please write your code here.
    int n;
    cin >> n;
    double scores_sum = 0.0, avg;
    vector<double> scores(n);

    for(int i = 0; i < n; i++)
    {
        cin >> scores[i];
    }

    scores_sum += accumulate(scores.begin(), scores.end(), 0.0);
    avg = scores_sum / n;

    cout << fixed << setprecision(1) << avg << endl;
    if(avg >= 4.0)
        cout << "Perfect";
    else if(avg >= 3.0)
        cout << "Good";
    else
        cout << "Poor";
    return 0;
}